foundations/powerPlatform/readme.md

```diff
@@ -222,6 +225,10 @@ The following section describes the design considerations and the design recomme
 - Enable auditing for your tenant and environments to understand usage and available capacity.
 - For Microsoft Cloud for industry solutions (e.g., D365 applications for Healthcare, Financial Services Industry), deploy all Dynamics 365 applications to the same environment(s), and avoid creating islands of data that will be complex to merge and combine later.
 - Only split Microsoft Cloud for industry solutions (D365 applications) across different environments if there are data or security constraints.
+- Use Managed Environments for the environments as it provides a suite of capabilities that allows admins to manage Power Platform at scale with more control, such as:
+  - Weekly digest, which informs about what's happening in the environment, such as analytics about your top apps and flows, your must impactful makes, and inactive resources you can safely clean up, deliverd to one or more mailboxes once a week.
+  - Sharing limits, where admins can limit how broadly makers can share canvas apps, by either excluding the sharing capability entirely, or define the limits of how many the apps can be shared with.
```

Comment:
Excluding the sharing capability entirely can currently be accomplished by setting the sharing restriction value to "1".  Was the intent here to state this or was it to state that excluding group-based sharing capability is possible. Either way we should align to the current documents: https://docs.microsoft.com/en-us/power-platform/admin/managed-environment-enable

Exclude sharing with security groups | Blocks the ability to share canvas apps with any security groups. Admins may share with a limit on who an app can be shared with

Limit total individuals who can be shared to | If Exclude sharing with security groups is selected, admin my limit the number of users a canvas app is shared with